ui/helpers/constants.js

```diff
@@ -9,6 +9,7 @@ export const thPlatformMap = {
   'linux32-devedition': 'Linux DevEdition',
   'linux32-nightly': 'Linux Nightly',
   'linux32-shippable': 'Linux shippable',
+  'linux1804-32-shippable': 'Linux 1804 shippable',
```

Comment:
I think adding a period to `18.04` would look nicer.

Reply:
Is there possibly a way I can mock this up either locally on a test server?

Reply:
We can launch it to prototype, but our docs also outline how to run the UI locally: https://treeherder.readthedocs.io/installation.html#ui-development

Is that what you meant?

Reply:
You can start the backend container with `docker-compose run backend bash`.
You can then run `./manage.py ingest task --task-id foo`.

On another tab either run `docker-compose up` or `yarn install && yarn run`. Look at the output and at some point you should be able to see the UI and show you whatever task you ingested.

Reply:
Shouldn't running `yarn start` work fine though @armenzg? This is just a mapping change in the UI and if he runs the UI against the production server then he should get all the data he needs?

Reply:
You're right. I forgot that the UI proxies.

@worldomonation what Sarah says ^

Reply:
> We can launch it to prototype if you push against the TH repo (I think also the heroku review app deployment works against our repo, not your fork). Otherwise our docs also outline how to run the UI locally: https://treeherder.readthedocs.io/installation.html#ui-development

Yes, that is what I meant. I will try running the local copy and see how the pretty names look like on it, and make changes as required. I'll ping once I am satisfied with the pretty name and ready for re-review.